pandas/core/series.py

```diff
@@ -3482,11 +3481,25 @@ def sort_values(
     ) -> None:
         ...
 
+    @overload
+    def sort_values(
```

Comment:
NDFrame declared this fallback overload, mypy seems to be stricter now and requires it also in the subclasses.